src/probe_matcher.cpp

```diff
@@ -212,8 +212,9 @@ std::unique_ptr<std::istream> ProbeMatcher::get_symbols_from_file(
 {
   auto file = std::make_unique<std::ifstream>(path);
   if (file->fail()) {
-    throw std::runtime_error("Could not read symbols from " + path + ": " +
-                             strerror(errno));
+    LOG(WARNING) << "Could not read symbols from " << path << ": "
+                 << strerror(errno);
+    return NULL;
```

Comment:
Nit: we prefer `nullptr` (even though the original code didn't respect that).
```suggestion
    return nullptr;
```